documentation/docs/examples/point-to-point-measurement.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: point-to-point-measurement
+title: Point to Point Measurement
+description: Demo shows how to measure point to point distance in Reveal.
+---
+
+import { DemoWrapper } from '@site/docs/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The `MeasurementTool` is a measuring tool for point to point distance with the 3D objects.
+This can be achieved by selecting a point on the 3D Object and drag the pointer to the destined point to get measurement of the distance.
+The tools default measurement is in meters, but the tool provides user to customise the measuring units.
+
+The tool provides ability to change measuring line width & color.
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+## Point to point distance measurment
+To enter into point to point measurement mode, use `MeasurementTool.enterMeasurementMode()` which will have default line width & color.
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+```
+### Exit measurement
+To exit from measurement mode, use `MeasurementTool.exitMeasurementMode()`.
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+measurementTool.exitMeasurementMode();
+```
+
+### Set width and color of the measuring line
+To change width and color of the line, use `MeasurementTool.setLineOptions(options: MeasurementOptions, meshObject?: THREE.Mesh)`. MeasurementOptions expects
+  - `lineWidth` : number
+  - `color` : number
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const lineOptions = {
+  lineWidth: 3,
+  color: 0xff0000
+};
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+measurementTool.setLineOptions(lineOptions);
```

Comment:
I'd switch these two. Does it even work to call `setLineOptions` after `enterMeasurementMode` after my changes?

Reply:
@larsmoa Below are different ways the tool supports

- `new MeasurementTool(viewer, options)` where options take `lineWidth` or/and `color` or/and `distanceToLabelCallback`.
- `MeasurementTool.setLineOptions(options)` here the next measuring line will have `options` `lineWidth` & `color`, the new measuring line.
- 'MeasurementTool.updateLineWidth(measurementId, lineWidth)` & `MeasurementTool.updateLineColor(measurementId, color)` where line width & color of certain measurement (identifed by Id) is update.

So do you expect the `setLineOptions` to consider both width & color than partial inputs?